app/src/main/res/layout/activity_loan_account_details.xml

```diff
@@ -15,6 +15,7 @@
         android:gravity="center_vertical"
         android:orientation="horizontal"
         android:paddingLeft="@dimen/row_padding_left"
```

Comment:
This line is redundant as the minApiVersion is 17. `android:paddingStart="@dimen/row_padding_left"` in itself is sufficient.